### ISSUE TITLE
Auth response returning wrong notification_cue_standards

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispersservices_django#336]
I am testing the standard notifications settings stuff and I submitted an update to the notifications setting for the admin user (per instructions from here:  https://github.com/USGS-WiM/whispers/issues/801) and it works. The response comes as expected (see image below).

![image](https://user-images.githubusercontent.com/6978380/75077819-723d4880-54c9-11ea-852b-2a3f7fa9f349.png)

I can also query the user directly and see that change reflected:
![image](https://user-images.githubusercontent.com/6978380/75078056-2ccd4b00-54ca-11ea-87d9-95420a533f90.png)

So far so good. But then for some reason my form was not reflecting that change when the page is reloaded. Checked and it was not an issue with the code. What I found is that my global `currentUser` object that I use to populate that form on page load with the user's preferences comes from the auth response on login, and is stored in session storage.That auth response has an inaccurate `notification_cue_standards` object:

![image](https://user-images.githubusercontent.com/6978380/75078269-cac11580-54ca-11ea-880a-5b7f883a4d9e.png)

**Comment by aaronstephenson** [comment]
It looks to me that what you're describing is stale data in your session storage. After your PATCH to `/users/1/` with altered `new_notification_cue_standard_preferences`, the data you're holding in session storage from the initial `auth` POST is now out of date. If you POST to `/auth/` again after that PATCH, the response is correct. Or, you could update your session storage data with the response from the PATCH.

I just performed a PATCH to `/users/1/` with altered `new_notification_cue_standard_preferences`, and then a POST to `/auth/`, and the response from both matched.

![image](https://user-images.githubusercontent.com/8713712/75082495-7a04e900-54d9-11ea-8578-7f34061f488b.png)


![image](https://user-images.githubusercontent.com/8713712/75082515-8f7a1300-54d9-11ea-9790-4492008bef18.png)

**Comment by BlakeDraper** [comment]
tl;dr: this is resolved.

But for the record, my testing on Friday was done with Insomnia which is not storing the user data in sessionStorage and you can see from those screencaps, the auth response data form the server was stale. I can't argue with the fact that it is working now, and I was aware of the sessionStorage factor in the client app even as I submitted this issue - the Insomnia response from the `auth` POST showing the outdated data was the smoking gun for me on Friday. Those 3 captures from my original post are in chronological sequence, so.... 🤷‍♂  At any rate - not sure what changed if you did not tweak the server in any way but I'll just chalk this up as a victory and move on.
